dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -74,7 +79,6 @@ interface RequestApi {
     /** Retrieves aggregated data requests by aggregating all userIds
      * @return aggregated data requests that match the given filters
      */
```

Comment:
One thing that came to my mind when testing the endpoints.
Currently it is not possible to update a request, correct? So if i request a dataset with the wrong contact email, iam currently not able to update it to the correct one. Do we want this?

Reply:
I think we will definitely want this. It shall generally be possible for a user to edit his/her sent requests, so this is something that needs to be implemented anyways. However, I am not sure whether it already sufficiently worked out how emails shall be sent in case of an updated request. In any case, I think this is nothing for this story

Reply:
Can you create a story for this or add it to an existing one if applicable? Once that is done i will resolve this here

Reply:
I added this to the description of https://jira.d-fine.dev/browse/DALA-1532

Reply:
ok, thank you